datasets/modis/modis-vegetation-example.ipynb

```diff
@@ -0,0 +1,776 @@
+      ],
+      "text/plain": [
+       "<xarray.DataArray '500m_16_days_NDVI' (time: 4, y: 570, x: 446)>\n",
+       "array([[[0.328 , 0.328 , 0.328 , ..., 0.1801, 0.1801, 0.1447],\n",
+       "        [0.3421, 0.3421, 0.3421, ..., 0.1793, 0.1793, 0.1659],\n",
+       "        [0.3475, 0.3723, 0.3772, ..., 0.1779, 0.1245, 0.1245],\n",
+       "        ...,\n",
+       "        [0.    , 0.    , 0.    , ..., 0.5779, 0.6112, 0.5442],\n",
+       "        [0.2205, 0.2205, 0.5283, ..., 0.5831, 0.5931, 0.6286],\n",
+       "        [0.    , 0.3147, 0.588 , ..., 0.6148, 0.6125, 0.6125]],\n",
+       "\n",
+       "       [[0.3048, 0.269 , 0.269 , ..., 0.1271, 0.1271, 0.5162],\n",
+       "        [0.2865, 0.2865, 0.269 , ..., 0.436 , 0.436 , 0.436 ],\n",
+       "        [0.3183, 0.2999, 0.3096, ..., 0.1186, 0.4259, 0.4259],\n",
+       "        ...,\n",
+       "        [0.    , 0.    , 0.    , ..., 0.4879, 0.4992, 0.3813],\n",
+       "        [0.    , 0.    , 0.591 , ..., 0.6288, 0.6409, 0.5544],\n",
+       "        [0.    , 0.    , 0.6003, ..., 0.6608, 0.6409, 0.6409]],\n",
+       "\n",
+       "       [[0.2843, 0.2435, 0.2435, ..., 0.6306, 0.6306, 0.6927],\n",
+       "        [0.2562, 0.2562, 0.2332, ..., 0.6466, 0.6466, 0.6927],\n",
+       "        [0.2556, 0.2541, 0.258 , ..., 0.5354, 0.5304, 0.6671],\n",
+       "        ...,\n",
+       "        [0.    , 0.    , 0.1783, ..., 0.2896, 0.3319, 0.3319],\n",
+       "        [0.2429, 0.2429, 0.    , ..., 0.4388, 0.4581, 0.4438],\n",
+       "        [0.    , 0.    , 0.4658, ..., 0.5297, 0.5027, 0.5027]],\n",
+       "\n",
+       "       [[0.444 , 0.4985, 0.4985, ..., 0.4981, 0.4981, 0.5898],\n",
+       "        [0.4498, 0.4498, 0.4985, ..., 0.6344, 0.6344, 0.6179],\n",
+       "        [0.4771, 0.4771, 0.4946, ..., 0.4196, 0.4147, 0.4147],\n",
+       "        ...,\n",
+       "        [0.2697, 0.2697, 0.2697, ..., 0.4864, 0.5066, 0.5138],\n",
+       "        [0.2697, 0.2697, 0.2697, ..., 0.5255, 0.5152, 0.5138],\n",
+       "        [0.3559, 0.3559, 0.3559, ..., 0.5572, 0.5692, 0.5692]]])\n",
+       "Coordinates:\n",
+       "  * time         (time) datetime64[ns] 2021-02-01T23:59:59.500000 ... 2021-10...\n",
+       "  * y            (y) float64 4.808e+06 4.807e+06 ... 4.524e+06 4.523e+06\n",
+       "  * x            (x) float64 -1.364e+07 -1.364e+07 ... -1.341e+07 -1.341e+07\n",
+       "    spatial_ref  int32 3857"
+      ]
+     },
+     "execution_count": 55,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import odc.stac\n",
+    "from pystac.extensions.raster import RasterExtension\n",
+    "\n",
+    "data = odc.stac.load(\n",
+    "    items.values(),\n",
+    "    crs=\"EPSG:3857\",\n",
```

Comment:
Can you share a bit (in the prose above) about how / why you picked this EPSG code? I see that `odc.stac.load` doesn't require this, but this is nicer for visualization?

Reply:
Yup, updated.